extensions/sequelize/src/sequelize/connector-mapping.ts

```diff
@@ -24,3 +24,55 @@ export const SupportedConnectorMapping: {
   sqlite3: 'sqlite',
   db2: 'db2',
 };
+
+/**
+ * Loopback uses different keys for pool options depending on the connector.
+ */
+export const poolConfigKeys = [
+  // mysql
+  'connectionLimit',
+  'acquireTimeout',
+  // postgresql
+  'min',
+  'max',
+  'idleTimeoutMillis',
+  // oracle
+  'minConn',
+  'maxConn',
+  'timeout',
+] as const;
+export type LoopbackPoolConfigKey = (typeof poolConfigKeys)[number];
+
+export type PoolingEnabledConnector = Exclude<
+  SupportedLoopbackConnectors,
+  'db2' | 'sqlite3'
+>;
+
+export const poolingEnabledConnectors: PoolingEnabledConnector[] = [
+  'mysql',
+  'oracle',
+  'postgresql',
+];
+
+type IConnectionPoolOptions = {
+  [connectorName in PoolingEnabledConnector]?: {
+    [sequelizePoolOption in keyof PoolOptions]: LoopbackPoolConfigKey;
+  };
+};
+
+export const ConnectionPoolOptions: IConnectionPoolOptions = {
```

Comment:
Should we not use strategy pattern here keeping the base options same ?

Reply:
True, even sequelize have a unified options interface across dialects, but this mapping above is added to support the existing configurations which users have been passing in loopback (juggler).

Different loopback connectors have different `keys` for setting up the connection pooling. For eg. mysql connector accepts `connectionLimit` whereas oracle connector takes in the same thing using `maxConn` key.

So If we don't do it this way, things would break if they use this sequelize datasource keeping their existing configuration in-tact.

Reply:
While it's not directly related this PR, I'm working on exposing a way for users to provide options directly to the sequelize. So users who are already comfortable using sequelize don't have to take care of such irregularities.

Reply:
ok understood